code/assets/css/form.css

```diff
@@ -0,0 +1,75 @@
+@import "./normalize.css";
+@import "./variables.css";
+@import "./general.css";
+@import "./animations.css";
+@import "./formpage-header.css";
+@import "./footer.css";
+
+.main-body {
+  background-color: #f1f1f1;
+}
+
+.form-container {
+  padding: calc(var(--spacing) * 4) 0;
+}
+
+.form {
+  overflow: hidden;
+}
+
+.form-title {
+  color: var(--primaryColorDarker);
+  font-size: var(--medium);
+  margin-bottom: calc(var(--spacing) * 4);
+}
+
+.fieldset {
+  background-color: var(--white);
+  margin-bottom: calc(var(--spacing) * 6);
+  border: 1px solid #ccc;
+}
+
+.map iframe {
+  width: 100%;
+  margin-top: calc(var(--spacing) * 2);
+}
+
+input:not([type="radio"]):not([type="checkbox"]),
+textarea {
+  display: block;
+  margin-bottom: calc(var(--spacing) * 4);
+  width: 100%;
+  padding: calc(var(--spacing) * 2);
+  /* border: 1px solid red; */
```

Comment:
Before final push always check your files and remove all commented code - needs to be fixed